netutils/webclient/webclient.c

```diff
@@ -740,6 +769,40 @@ int webclient_perform(FAR struct webclient_context *ctx)
               return -errno;
             }
 
+#if defined(CONFIG_WEBCLIENT_NET_LOCAL)
+          if (ctx->unix_socket != NULL)
+            {
+              memset(&server_un, 0, sizeof(server_un));
+              server_un.sun_family = AF_LOCAL;
+              strncpy(server_un.sun_path, ctx->unix_socket,
+                      sizeof(server_un.sun_path));
+#if !defined(__NuttX__) && !defined(__linux__)
+              server_un.sun_len = SUN_LEN(&server_un);
```

Comment:
BTW, do we really need set sun_len on macOS?

Reply:
yes